src/main/resources/liquibase/crop_changelog/13_2_0.xml

```diff
@@ -486,5 +486,74 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="vanina" id="v13.2.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="stock_id_bkp"></tableExists>
+			</not>
+		</preConditions>
+		<comment>
+			Create stockId backup table and migrate inventory_id values for more than 1 stock_id
```

Comment:
Comment is wrong because we are storing all inventory_ids except by the null ones

Reply:
Ok